recipes/eigen/meta.yaml

```diff
@@ -0,0 +1,32 @@
+{% set version = "3.2.8" %}
+
+package:
+  name: eigen
+  version: {{ version }}
+
+source:
+  fn: eigen-{{ version }}.tar.gz
+  url: http://bitbucket.org/eigen/eigen/get/{{ version }}.tar.gz
+  sha1: 574b0732f7e04ecfa91597afb2f69bc44f3387e6
+
+requirements:
+  build:
+    - cmake
+
+test:
+  files:
+    - test.cc
+
+  commands:
+    - test -d $PREFIX/include/eigen3  # [unix]
+    - if not exist %PREFIX%\\Library\\include\\Eigen exit 1  # [win]
```

Comment:
As this is an includes only library, just verifies the includes our present. Thanks @patricksnape.